glean-core/ios/Glean/Glean.swift

```diff
@@ -25,7 +25,7 @@ public class Glean {
 
     var handle: UInt64 = 0
     private var uploadEnabled: Bool = true
-    private var configuration: Configuration?
+    var configuration: Configuration?
```

Comment:
Why does this need to become non-private?

Reply:
I needed to change this to `internal` so that I can check values in a test to verify that the `handleCustomUrl` function is properly setting the `pingTag` and `logPings` values.  

In relaxing this to `internal` access, this makes it in line with the Android configuration instance in the Glean object, which is also listed as internal.